### PR TITLE
En 10606 review fixes

### PR DESCRIPTION
**Pull request by mariusmihaic**
Added all fixes from previous PRs here

### CODE REVIEW

File: process/transactions/transactions_processor_test.go [hunk 1]
```diff
@@ -305,20 +310,20 @@ func requireProcessedTransactionEqual(
...
+	require.Equal(t, utility.EncodePubKey(pubKeyConverter, tx.GetRcvAddr()), processedTx.Receiver)
+	require.Equal(t, utility.EncodePubKey(pubKeyConverter, tx.GetSndAddr()), processedTx.Sender)
+	require.Equal(t, int32(miniBlock.GetReceiverShardID()), processedTx.ReceiverShard)
+	require.Equal(t, int32(miniBlock.GetSenderShardID()), processedTx.SenderShard)
+	require.Equal(t, int64(tx.GetGasPrice()), processedTx.GasPrice)
+	require.Equal(t, int64(tx.GetGasLimit()), processedTx.GasLimit)
+	require.Equal(t, tx.GetSignature(), processedTx.Signature)
+	require.Equal(t, tx.GetSndUserName(), processedTx.SenderUserName)
+	require.Equal(t, tx.GetRcvUserName(), processedTx.ReceiverUserName)
 	require.Equal(t, processedTx.MiniBlockHash, mbHash)
```
Comment (AdoAdoAdo) [inline process/transactions/transactions_processor_test.go]: I think the expected value is mbHash

File: process/accounts/accounts_processor.go [hunk 2]
```diff
@@ -94,6 +98,8 @@ func (ap *accountsProcessor) addAddressIfInSelfShard(addresses map[string]struct
 }
 
 func (ap *accountsProcessor) processAccount(address string) (*schema.AccountBalanceUpdate, error) {
+	//TODO: This only works as long as covalent indexer is part of elrond binary client.
```
Comment (AdoAdoAdo) [inline process/accounts/accounts_processor.go]: please change "is part of elrond binary client" to "is part of elrond node binary"

File: process/transactions/transactions_processor_test.go (hunk 1 above)
Comment (mariusmihaic) [reply process/transactions/transactions_processor_test.go]: Yep. I've missed this one. Fixed it

File: process/accounts/accounts_processor.go (hunk 2 above)
Comment (mariusmihaic) [reply process/accounts/accounts_processor.go]: Changed it !

**Review by AdoAdoAdo — APPROVED** [review]

File: process/interface.go [hunk 3]
```diff
@@ -25,9 +25,9 @@ type TransactionHandler interface {
 		transactions map[string]data.TransactionHandler) ([]*schema.Transaction, error)
 }
 
-// SCHandler defines what a smart contract processor shall do
-type SCHandler interface {
-	ProcessSCs(transactions map[string]data.TransactionHandler, timeStamp uint64) []*schema.SCResult
+// SCResultsHandler defines what a smart contract processor shall do
```
Comment (miiu96) [inline process/interface.go:28]: // SCResultsHandler defines what a smart contract results processor shall do

File: process/transactions/scr_processor.go [hunk 4]
```diff
@@ -14,18 +14,18 @@ type scProcessor struct {
...
 		return nil, covalent.ErrNilPubKeyConverter
 	}
 
 	return &scProcessor{pubKeyConverter: pubKeyConverter}, nil
 }
 
-// ProcessSCs converts smart contracts data to a specific structure defined by avro schema
-func (scp *scProcessor) ProcessSCs(transactions map[string]data.TransactionHandler, timeStamp uint64) []*schema.SCResult {
-	allSCRs := make([]*schema.SCResult, 0)
+// ProcessSCRs converts smart contracts data to a specific structure defined by avro schema
```
Comment (miiu96) [inline process/transactions/scr_processor.go:26]: // ProcessSCRs converts smart contract results data to a specific structure defined by avro schema

**Review by miiu96 — APPROVED** [review]

File: process/interface.go (hunk 3 above)
Comment (mariusmihaic) [reply process/interface.go:28]: Will change it in latest PR

File: process/transactions/scr_processor.go (hunk 4 above)
Comment (mariusmihaic) [reply process/transactions/scr_processor.go:26]: Will change it in latest PR

**Pull request merged**